### ISSUE TITLE
Create an endpoint for the contact form on the McKenzie site

### ISSUE THREAD

**Issue by RyanPnayR** [RyanPnayR/issue-migration#9]
In gitlab by @randseay on Sep 4, 2017, 14:41

It makes sense for all of the fields to be required. If I could get a response with a success/error indicator, that would be great. I would like to `post` using `fetch()`, which would send this `json` response:

```json
{
    email: string;
    name: string;
    phone: string;
    text: string;
}
```

**Comment by RyanPnayR** [comment]
In gitlab by @jeffrey.a.barron on Sep 4, 2017, 19:08

mentioned in commit 2405f7b988f32718f5c65e7b762ae6b49938d979

**Comment by RyanPnayR** [comment]
In gitlab by @jeffrey.a.barron on Sep 4, 2017, 19:09

mentioned in merge request !3

**Comment by RyanPnayR** [comment]
In gitlab by @jeffrey.a.barron on Sep 4, 2017, 19:12

@randseay I have it returning 400 for errors and 200 for success when posting to /api/contact.  This won't send email just yet - need to figure out how to integrate with Mailgun.

**Comment by RyanPnayR** [comment]
In gitlab by @jeffrey.a.barron on Sep 4, 2017, 19:13

added 2h 15m of time spent

**Comment by RyanPnayR** [comment]
In gitlab by @jeffrey.a.barron on Sep 4, 2017, 19:48

@randseay Added a toggle for you to test conditions via the post JSON: {"test_condition": "failure"} will throw a 400.  A missing `test_condition` or one with another value will use 200.

**Comment by RyanPnayR** [comment]
In gitlab by @randseay on Sep 5, 2017, 23:37

@jeffrey.a.barron jeff sounds great, nice thinking with the `test_condition`.

**Comment by RyanPnayR** [comment]
In gitlab by @nZac on Oct 4, 2017, 09:27

closed via commit 653e75455eac2998f5108eaae3ab130a165ef462
